azext_edge/edge/providers/check/common.py

```diff
@@ -126,6 +168,18 @@ class DataprocessorDestinationStageType(ListableEnum):
     DataprocessorDestinationStageType.reference_data.value: [("dataset", "Dataset ID", False)]
 }
 
+DATA_PROCESSOR_AUTHENTICATION_REQUIRED_PROPERTIES = {
+    DataprocessorAuthenticationType.accessToken.value: ["accessToken"],
+    DataprocessorAuthenticationType.accessKey.value: ["accessKey"],
+    DataprocessorAuthenticationType.header.value: ["key", "value"],
+    DataprocessorAuthenticationType.metadata.value: ["key", "value"],
+    DataprocessorAuthenticationType.none.value: [],
+    DataprocessorAuthenticationType.serviceAccountToken.value: [],
+    DataprocessorAuthenticationType.servicePrincipal.value: ["tenantId", "clientId", "clientSecret"],
+    DataprocessorAuthenticationType.systemAssignedManagedIdentity.value: [],
```

Comment:
is this supposed to be empty? usually there is a principal id and some other id

Reply:
yeah only the type itself is needed, no other properties according to service spec